pandas/core/indexes/multi.py

```diff
@@ -1035,11 +1035,13 @@ def is_lexsorted(self):
         """
         return self.lexsort_depth == self.nlevels
 
-    def is_lexsorted_for_tuple(self, tup):
+    def _true_slices_indices(self, tup):
```

Comment:
I need to change the method (not just the name, at least the signature), because I need "last_slice" for the error message. I can just suppress it if you prefer.